### PR TITLE
Issue #3269254 by DamienMcKenna, tbsiqueira, navneet0693: Upgrade to core 9.3.x

### PR DESCRIPTION
**Pull request by tbsiqueira**
## Problem
Drupal 9.2 is reaching its EOL on June 15th.

## Solution
We need to update Open Social dependency to support Drupal Core 9.3.

https://www.drupal.org/project/drupal/releases/9.3.0

This PR is one of the PRs originated from #2748, we decided to split it into minor PRs to better test all the changes.

## Issue tracker
- https://www.drupal.org/project/social/issues/3269254
- https://www.drupal.org/project/social/issues/3282073

## Theme issue tracker
N/A

## How to test
- [ ] Install this branch;
- [ ] Smoke test all system and make sure all test scenarios are still passing;

### Before merge
- [ ] Code/peer review is completed
- [ ] All commit messages are [clear and clean](https://open-social.slite.com/app/docs/DnmermZDIx_0OQ). If applicable a rebase was performed
- [ ] All automated tests are green
- [ ] Functional/manual tests of the acceptance criteria are approved
- [ ] All acceptance criteria were met
- [ ] If applicable (I.E. new hook_updates) the update path from previous versions (major and minor versions) are tested
- [ ] This pull request has all required labels (team/type/priority)
- [ ] This pull request has a milestone
- [ ] This pull request has an assignee (if applicable)
- [ ] Any front end changes are tested on all major browsers
- [ ] New UI elements, or changes on UI elements are approved by the design team
- [ ] New features, or feature changes are approved by the product owner

### After merge
- [ ] Code is tested on all branches that it has been cherry-picked
- [ ] Update hook number might need adjustment, make sure they have the correct order
- [ ] The Drupal.org ticket(s) are updated according to this pull request status

## Screenshots
N/A

## Release notes
N/A

## Change Record

- https://www.drupal.org/node/3281588
- https://www.drupal.org/node/3281590
- https://www.drupal.org/node/3281587
- https://www.drupal.org/node/3281583
- https://www.drupal.org/node/3281582

## Translations
N/A

### CODE REVIEW

**Comment by open-social-tugboat** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr2919-m0zy94jezi4ryeybfv92463doxoqfadc.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/62712ec6ef17c6f569a7bdbd

**Comment by open-social-tugboat** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr2919-m0zy94jezi4ryeybfv92463doxoqfadc.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/62712ec6ef17c6f569a7bdbd

**Comment by open-social-tugboat** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr2919-m0zy94jezi4ryeybfv92463doxoqfadc.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/62712ec6ef17c6f569a7bdbd

**Comment by open-social-tugboat** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr2919-m0zy94jezi4ryeybfv92463doxoqfadc.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/62712ec6ef17c6f569a7bdbd

**Comment by open-social-tugboat** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr2919-m0zy94jezi4ryeybfv92463doxoqfadc.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/62712ec6ef17c6f569a7bdbd

**Comment by open-social-tugboat** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr2919-m0zy94jezi4ryeybfv92463doxoqfadc.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/62712ec6ef17c6f569a7bdbd

**Comment by open-social-tugboat** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr2919-m0zy94jezi4ryeybfv92463doxoqfadc.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/62712ec6ef17c6f569a7bdbd

**Comment by open-social-tugboat** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr2919-m0zy94jezi4ryeybfv92463doxoqfadc.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/62712ec6ef17c6f569a7bdbd

**Comment by open-social-tugboat** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr2919-m0zy94jezi4ryeybfv92463doxoqfadc.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/62712ec6ef17c6f569a7bdbd

**Comment by ronaldtebrake** [comment]
Is there still something we need to do for:
https://www.drupal.org/node/3158256?

**Comment by ronaldtebrake** [comment]
Screenshot 2022-05-05 at 12 34 22

This seems related to the update (?)

File: composer.json
Comment (tbsiqueira) [reply composer.json:226]: Ronald I added to the commit message https://github.com/goalgorilla/open_social/pull/2919/commits/6280273a70c95d38e47a69038fed4b8838ac9a07 maybe we should add to another channel?

**Comment by open-social-tugboat** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr2919-m0zy94jezi4ryeybfv92463doxoqfadc.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/62712ec6ef17c6f569a7bdbd

**Comment by open-social-tugboat** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr2919-m0zy94jezi4ryeybfv92463doxoqfadc.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/62712ec6ef17c6f569a7bdbd

**Comment by open-social-tugboat** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr2919-m0zy94jezi4ryeybfv92463doxoqfadc.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/62712ec6ef17c6f569a7bdbd

**Comment by open-social-tugboat** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr2919-m0zy94jezi4ryeybfv92463doxoqfadc.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/62712ec6ef17c6f569a7bdbd

**Comment by open-social-tugboat** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr2919-m0zy94jezi4ryeybfv92463doxoqfadc.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/62712ec6ef17c6f569a7bdbd

**Comment by open-social-tugboat** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr2919-m0zy94jezi4ryeybfv92463doxoqfadc.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/62712ec6ef17c6f569a7bdbd

**Comment by open-social-tugboat** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr2919-m0zy94jezi4ryeybfv92463doxoqfadc.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/62712ec6ef17c6f569a7bdbd

**Comment by open-social-tugboat** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr2919-m0zy94jezi4ryeybfv92463doxoqfadc.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/62712ec6ef17c6f569a7bdbd

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr2919-m0zy94jezi4ryeybfv92463doxoqfadc.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/62712ec6ef17c6f569a7bdbd

File: composer.json
Comment (ronaldtebrake) [reply composer.json:221]: Basically we should check if we should add it to a different channel, we used to lock the version to `"swiftmailer/swiftmailer": "6.2.4"`
however, by removing the dependency here, we are now using the dependency from 
https://git.drupalcode.org/project/swiftmailer/-/blob/8.x-2.x/composer.json#L11

Which isn't locking the version, so it's likely it will resolve to a different version now
Your commit message clearly explains why, but not what version we now upgrade/downgraded to.

**Comment by ronaldtebrake** [comment]
Seems like most of my remarks are taken care off, lets see what Travis thinks and maybe good to get another pair of @goalgorilla/maintainers eyes on this :)

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr2919-m0zy94jezi4ryeybfv92463doxoqfadc.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/62712ec6ef17c6f569a7bdbd

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr2919-m0zy94jezi4ryeybfv92463doxoqfadc.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/62712ec6ef17c6f569a7bdbd

File: composer.json
Comment (tbsiqueira) [reply composer.json:221]: Done, added as a change record.

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr2919-m0zy94jezi4ryeybfv92463doxoqfadc.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/62712ec6ef17c6f569a7bdbd

**Comment by tbsiqueira** [comment]
We need to merge and approve https://github.com/goalgorilla/drupal_social/pull/372 after this PR is merged

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr2919-m0zy94jezi4ryeybfv92463doxoqfadc.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/62712ec6ef17c6f569a7bdbd

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr2919-m0zy94jezi4ryeybfv92463doxoqfadc.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/62712ec6ef17c6f569a7bdbd

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr2919-m0zy94jezi4ryeybfv92463doxoqfadc.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/62712ec6ef17c6f569a7bdbd

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr2919-m0zy94jezi4ryeybfv92463doxoqfadc.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/62712ec6ef17c6f569a7bdbd

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr2919-m0zy94jezi4ryeybfv92463doxoqfadc.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/62712ec6ef17c6f569a7bdbd

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr2919-m0zy94jezi4ryeybfv92463doxoqfadc.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/62712ec6ef17c6f569a7bdbd

**Review by navneet0693 — APPROVED** [review]

**Pull request merged**
